cargo_van.rb

```diff
@@ -1,9 +1,51 @@
 require_relative 'van.rb'
 
 class CargoVan < Van
+
+  attr_reader :total_volume
   
-  def initialize(number, type = 'cargo')
-    super
+  def initialize(total_volume)
+    @number = generate_number
+    @total_volume = total_volume
+    validate!
+    @volume = 0
+  end
+
+  def increase_volume(volume)
+    return if self.volume + volume > self.total_volume
```

Comment:
для ридеров "селф" писать не рекомендуют, для сеттеров он обязательный
здеьс лучше так:
```suggestion
    return if volume + volume > total_volume
```
но так будет путанница с именами
поэтому лучше воспользоваться геттером, который ты написала (28 строка)
```suggestion
    return if occupied_volume + volume > total_volume
```
еще можно воспользоваться методом `free_volume` и тогда запись станет еще понятней и лаконичней
```suggestion
    return if free_volume < volume
```